qiskit/execute.py

```diff
@@ -19,8 +19,10 @@
 import logging
 import warnings
 
-from qiskit.compiler import assemble_circuits, assemble_schedules, transpile
 from qiskit.compiler import RunConfig, TranspileConfig
+from qiskit.compiler import assemble_circuits, assemble_schedules, transpile
+from qiskit.exceptions import QiskitError
+from qiskit.pulse import Schedule, ConfiguredSchedule, ScheduleConfig
```

Comment:
Can you move `ScheduleConfig` to `qiskit.compiler.schedule_config` ? Then we can import it as
```
from qiskit.compiler import RunConfig, TranspileConfig, ScheduleConfig
```
This is more aligned with how `QuantumCircuit` is assembled.